pkg/kubelet/volume/reconciler/reconciler.go

```diff
@@ -199,6 +243,16 @@ func (rc *reconciler) reconciliationLoopFunc() func() {
 						attachedVolume.VolumeSpec.Name())
 					err := rc.operationExecutor.UnmountDevice(
 						attachedVolume.AttachedVolume, rc.actualStateOfWorld)
+					if !goroutinemap.IsAlreadyExists(err) {
```

Comment:
You should check `err != nil` here too, or you'll log error messages for nil errors.